### DISCUSSION TITLE
Quasar v2 stable released! What does this mean? (please read)

### DISCUSSION THREAD

**Discussion by rstoenescu** [quasarframework/quasar#9734] [Announcements]
Quasar v2 stable (**with Vue.js 3**) has been released!

This brings a few changes:
1. https://quasar.dev now points to the Quasar v2 docs. For Quasar v1 docs, https://v1.quasar.dev will be used instead.
2. The `quasar create` command now generates a Qv2 project by default. For Qv1 projects you'll have to use `quasar create <folder> --branch v1`.
3. On this repository: the `dev` branch will be used exclusively for Quasar v2; for Quasar v1, I've created a new branch called `v1`. The `vue3-work` branch will no longer exist.
4. Quasar v1 will still receive updates (and backports that can fit) until EOL 2022/04/01.

There's a huge list of thanks that we need to give out:
* Our sponsors and donators. You keep us alive!
* Our early Qv2 adopters. You helped us reach the stable state much earlier!
* @yusufkandemir who kicked off the Vue 3 work
* The whole Quasar Team for its inhuman effort!
* And the last but not least, the Vue Team for Vue 3!

## Donations

If you want to speed up the development of Quasar please consider donating to the project. With proper funding, it allows more of the team to work on the project in a much more dedicated manner.

[Donations - https://donate.quasar.dev](https://donate.quasar.dev)

If you're in a company and using Quasar for commercial projects, explain to your management the importance of monthly donations (eg. $200+) for open source projects: you're the one using it every day and this makes you the best suited person to convince them. Be creative! :slight_smile:

Quasar is saving your company tens of thousands of development hours (quite literally), which in turn is a pretty high money savings. Consider giving back a part of those savings to refuel the project itself :wink:

**Comment by donPuerto** [comment]
Congratulations to @rstoenescu and the quasar team and to my homeboy @metalsadman ;)

**Comment by jeroenqui** [comment]
Congratulations and massive props on the amazing work done, on such a short term!

**Comment by 0xDevansh** [comment]
Amazing!!
Goodbye --branch next, you won't be missed

**Comment by wsw70** [comment]
Many people in the discussions are professional developers - I would like to voice out the **THANK YOU** from those such as myself that are amateur (home) developers and Quasar makes any front-end **immensely easier** to create and maintain.

So thank you very, very much!

**Comment by webnoob** [comment]
Awesome news! Well done everyone!

**Comment by sebaguse** [comment]
Best news of today, thank you for all your hard work and such an amazing framework you created! 👍

**Comment by Irev-Dev** [comment]
👏

Upgrade guide for anyone else looking for it.

https://quasar.dev/start/upgrade-guide

**Comment by natxocc** [comment]
Really great!!! 
Amount of tones of hours dedicated and shared with rest of people.
Not only with free of use, also with a high level of program quality, refined and numerous details that high factory's doesn't feed the same level.

Congratulations to rstoenescu and the big great team that are around this great project!

Of course.... **REALLY THANKS FOR ALL OF YOU!**

**Comment by Ajitweb** [comment]
**Congratulations to the entire Quasar team !**

**Comment by aircjm** [comment]
🎉🎉🎉🎉

**Comment by bichikim** [comment]
Oh yeah~~~
🎉🎉

**Comment by tmezzena** [comment]
Congratulations to the team and thanks to this awesome project

**Comment by herbnspice** [comment]
Congratulations to the team!

**Comment by smolinari** [comment]
This is a major milestone in Quasar's history. Awesome work Razvan and team. 

Scott

**Comment by ionesculiviucristian** [comment]
Buna treaba 🎉

**Comment by Lietsaki** [comment]
Congratulations. This is amazing 🎉

**Comment by SRIRAM301090** [comment]
Congratulation Quasar Team for this amazing framework

**Comment by JokerMartini** [comment]
Amazing stuff!

**Comment by robert-niestroj** [comment]
Hi,
since Vue 2.7 will EOL at the end of 2023 will Qv1 adapt to it and also LTS at the same day? That would mean a year longer then now planned.
https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule
https://blog.vuejs.org/posts/vue-2-7-naruto.html
